architecture/control-plane.adoc

```diff
@@ -44,8 +44,6 @@ include::modules/arch-platform-operators.adoc[leveloffset=+2]
 * xref:../installing/overview/cluster-capabilities.adoc#cluster-capabilities[Cluster capabilities]
 endif::[]
 
-include::modules/understanding-machine-config-operator.adoc[leveloffset=+1]
-
```

Comment:
We should remove the additional references or move them to the new location.

Reply:
Also up around line 15, these seem like they're not core control-plane issues:

```
include::modules/architecture-machine-config-pools.adoc[leveloffset=+1]
[role="_additional-resources"]
.Additional resources
* xref:../post_installation_configuration/machine-configuration-tasks.adoc#machine-config-drift-detection_post-install-machine-configuration-tasks[Understanding configuration drift detection].

include::modules/architecture-machine-roles.adoc[leveloffset=+1]
```

Reply:
@mburke5678 WDYT?

Reply:
@wking The link to _Understanding configuration drift detection_ is related to the previous paragraph, where we introduce the concept of config drift. As for the inclusion of architecture-machine-roles.adoc, it appears that was [added 5 years ago](https://github.com/openshift/openshift-docs/commit/630b9a5a313eae683d94ec69996bb7fa059e749c#diff-dfa72e2ab79511e5580b4ad769004040f37752d8bbcb8952db6b2e9c0f57914a). Not sure if @kalexand-rh  would remember why it was added. 
Can we pursue this through a separate issue/PR, so we can move this one along?

Reply:
@wking already let me know in a DM yesterday that he was fine if we progressed on this one and had a separate PR for this. :) I'll mark it as SME-approved and pass it along for peer review. TYSM! @mburke5678 can you create the ticket for the follow-up?

Reply:
Follow up: https://issues.redhat.com/browse/OCPBUGS-45992